src/api/models.py

```diff
@@ -2,18 +2,256 @@
 
 db = SQLAlchemy()
 
-class User(db.Model):
+clientfavcenter = db.Table('clientfavcenter',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('center_id', db.Integer, db.ForeignKey('center.id'), primary_key=True)
+)
+
+
+clientfavspot = db.Table('clientfavspot',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('hotspot_id', db.Integer, db.ForeignKey('hotspot.id'), primary_key=True)
+)
+
+
+reviewspot = db.Table('reviewspot',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('hotspot_id', db.Integer, db.ForeignKey('hotspot.id'), primary_key=True)
+)
+
+
+reviewcenter = db.Table('reviewcenter',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('center_id', db.Integer, db.ForeignKey('center.id'), primary_key=True)
+)
+
+
+clientsport = db.Table('clientsport',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('sport_id', db.Integer, db.ForeignKey('sport.id'), primary_key=True)
+)
+
+
+centersport = db.Table('centersport',
+    db.Column('center_id', db.Integer, db.ForeignKey('center.id'), primary_key=True),
+    db.Column('sport_id', db.Integer, db.ForeignKey('sport.id'), primary_key=True)
+)
+
+
+
+class Client(db.Model):
+    __tablename__: "client"
+
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(120), unique=True, nullable=False)
-    password = db.Column(db.String(80), unique=False, nullable=False)
-    is_active = db.Column(db.Boolean(), unique=False, nullable=False)
+    _password = db.Column(db.String(120), unique=False, nullable=False)
+    clientname = db.Column(db.String(120), unique=True, nullable=False)
+    name = db.Column(db.String(120), unique=False, nullable=False)
```

Comment:
Cual es la diferencia entre client name y name?

Reply:
client name = username
name = real name and surname